CHANGELOG.md

```diff
@@ -13,24 +13,35 @@
 #### Emoji Reactions
 - View Emoji Reactions on a post
 
+#### New Conversation
+- Start a new Direct Message or a Group Message from the same screen
+
+#### No Missed Messages
+- Auto-retry has been added so you won't ever miss a message
```

Comment:
0/5 where it goes, but is something like super important hehehehe.

What it does is that when you are in a poor connection sometimes getting the new posts when you switch to a channel may fail, so this will retry up to two times to get the new posts after that if it still failed then you'll be presented with a banner asking you to pull to refresh